services/src/main/scala/cromwell/services/metadata/impl/archiver/ArchiveMetadataConfig.scala

```diff
@@ -31,6 +33,8 @@ object ArchiveMetadataConfig {
 
   def parseConfig(archiveMetadataConfig: Config)(implicit system: ActorSystem): Checked[ArchiveMetadataConfig] = {
     val defaultMaxInterval: FiniteDuration = 5 minutes
+    val defaultArchiveDelay = 365 days
+    val defaultDeleteDelay  = 400 days
```

Comment:
Maybe worth a tech talk. I put all delete options into a separate stanza (and with separate parsing logic) in https://github.com/broadinstitute/cromwell/pull/6232 (https://github.com/broadinstitute/cromwell/pull/6232/files#diff-2b7da379864a890fadbd58178c6b69660d6b516f00ab025021961c2aa2de5ab5R19)

Reply:
Done.